src/platform/test/functional/services/combo_box.ts

```diff
@@ -38,6 +38,19 @@ export class ComboBoxService extends FtrService {
     await this.setElement(comboBox, value);
   }
 
+  /**
+   * Finds combobox element and sets specified value with retry logic for stale element handling
+   *
+   * @param comboBoxSelector data-test-subj selector
+   * @param value option text
+   */
+  public async setWithRetry(comboBoxSelector: string, value: string): Promise<void> {
```

Comment:
Sorry, one more question, is `setWithRetry` even needed? Could retry just be added to the existing `set` method?